starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +16,45 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(target){
+  var sorted = target.sort(
+    function(a, b){
+      return b.length - a.length;}
+  );
+  return sorted[0]
+};
+
+findLongestWord(words);
 
+
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+var sum = 0;
+function sumArray(target){
+  var result = target.reduce(function(accumulator, currentValue, currentIndex, array) {
```

Comment:
To resolve the error in jasmine just add this two lines of code : 

```js

  if (target.length == 0) {
    return sum;
  }

```

Because the jasmine want's you to deal with special case like this (empty array, ect..)